docs/vz235_mellow/top_cover/top_cover.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
+parent: 8. Top cover and doors panels
+grand_parent: Vz235 - Mellow Kit
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
+---
+
+# 8.1 Top Cover Assembly
+
+## Overview
+
+![Overview](../../assets/images/manual/vz235_printed/top_cover/top_cover_overview.png)
```

Comment:
In general, all the URL's for assets can be referred to from the root directory of the docs folder we build from (./docs from repo root, i.e. the second docs folder).

Relative URL's are not really necessary here; it may also increase the risk of frustration with the URL's, and can make the source code untidy when they become six or seven iterations deep. 😳

Reply:
I hadn't bothered changing the image that was already on the page. I may just remove the overview image entirely.
For the URL's I was just replicating what was already in use on the page.

Reply:
I have removed the overview image and left the URL's as they as as thats the way they are done on every other page I looked at